sdcm/db_stats.py

```diff
@@ -334,6 +334,24 @@ def get_scylla_scheduler_runtime_ms(self, start_time, end_time, node_ip, irate_s
                                                     [float(runtime[1]) for runtime in item['values']]})
         return res
 
+    def get_scylla_io_queue_total_operations(self, start_time, end_time, node_ip, irate_sample_sec='30s'):
+        """
+        Get Scylla CPU scheduler runtime from PrometheusDB
+
+        :return: list of tuples (unix time, op/s)
+        """
```

Comment:
I'm guessing yes because graphs look quite the same for scylla_scheduler_runtime_ms and scylla_io_queue_total_operations, just different values

![image](https://github.com/user-attachments/assets/180b33e1-4bb9-4e61-890f-c4ec3e9e7051)
![image](https://github.com/user-attachments/assets/27fc7c85-3a14-4951-99dd-cd300598c360)